video/out/vulkan/malloc.c

```diff
@@ -432,3 +450,9 @@ bool vk_malloc_buffer(struct mpvk_ctx *vk, VkBufferUsageFlags bufFlags,
 
     return true;
 }
+
+size_t vk_memslice_get_slab_size(struct mpvk_ctx *vk, struct vk_memslice *slice)
```

Comment:
Instead of a function for this, it would make more sense IMO to just add a field to vk_memslice. We already encode the offset relative to the underlying allocation, might as well encode its size as well - if use cases like this benefit from it.

Reply:
Done